extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcConstants.java

```diff
@@ -73,4 +73,18 @@ public final class OidcConstants {
     public static final String ID_TOKEN_SID_CLAIM = "sid";
 
     public static final String OPENID_SCOPE = "openid";
+
+    /**
```

Comment:
@michalvavrik This is a minor one, would you be OK with creating say `TokenPropagationConstants.java` in here:
https://github.com/quarkusio/quarkus/tree/main/extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/token/propagation

and refer to this class instead ? I've meant to keep `OidcConstants` as a reference to various standard OIDC scopes, properties, etc. 
(minor typo with `iff`)

Reply:
`iff` =>  if and only if :-) but I'll go with if
It means this constant won't be available in OIDC auth mechanism, but that's not a problem as there are tests and it is same situation as with JWT auth mechanism, I'll do it.